src/utils/wallet.js

```diff
@@ -458,7 +455,7 @@ const wallet = {
    * @inner
    */
   async resetWalletData() {
```

Comment:
Do you still need this method? I saw that you are swapping all the places to use the redux action instead.

Reply:
You're right, we can remove it

Fixed in be94f7c3f44c614130b263300673f8c63a4d200a